packages/main/test/pages/AvatarGroup.html

```diff
@@ -214,6 +214,78 @@ <h3>Unordinary group</h3>
 			<ui5-avatar size="XL" interactive initials="XL"></ui5-avatar>
 		</ui5-avatar-group>
 	</div>
+	<h1>AvatarGroup with totalAvatarCount smaller than ui5-avatar count used in items slot</h1>
```

Comment:
correct the section title together with the merge conflict resolution, because it shows the correct use-case